hpx/components/containers/partitioned_vector/partitioned_vector.hpp

```diff
@@ -732,6 +732,18 @@ namespace hpx
         }
 
     public:
+
+        std::vector< hpx::id_type > get_partitions_ids()
+        {
+            std::vector< hpx::id_type > ids;
+
+            for(auto const part_data : partitions_)
```

Comment:
Hmmm, the idea was for the `array_view` to access the local data only. Why do you need to get a copy of the local data for this? Wouldn't it be no _view_ anymore?